website/routes.py

```diff
@@ -1168,6 +1168,12 @@ def make_url_map(app):
             addon_views.addon_view_or_download_file,
             OsfWebRenderer('project/view_file.mako', trust=False)
         ),
+        Rule(
+            '/<pid>/download/<fid_or_guid>/',
+            'get',
+            addon_views.persistent_file_download,
+            json_renderer,
+        ),
```

Comment:
Ideally, the persistent download link should be addressed in V2 API instead of building up another V1 API.

Reply:
Talked with @brianjgeiger about this as that's what I first did. He said that these base osf routes would be guaranteed where as API routes would not be.